src/amo/pages/Feedback/index.js

```diff
@@ -0,0 +1,478 @@
+/* @flow */
+/* global window */
+import invariant from 'invariant';
+import * as React from 'react';
+import Textarea from 'react-textarea-autosize';
+import { Helmet } from 'react-helmet';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import config from 'config';
+
+import NotFoundPage from 'amo/pages/ErrorPages/NotFoundPage';
+import Page from 'amo/components/Page';
+import { sendAddonAbuseReport } from 'amo/reducers/abuse';
+import { getCurrentUser } from 'amo/reducers/users';
+import { VIEW_CONTEXT_HOME } from 'amo/constants';
+import { withFixedErrorHandler } from 'amo/errorHandler';
+import log from 'amo/logger';
+import translate from 'amo/i18n/translate';
+import Button from 'amo/components/Button';
+import Card from 'amo/components/Card';
+import Notice from 'amo/components/Notice';
+import type { AddonAbuseState } from 'amo/reducers/abuse';
+import type { UserType } from 'amo/reducers/users';
+import type { AppState } from 'amo/store';
+import type { ElementEvent, HTMLElementEventHandler } from 'amo/types/dom';
+import type { ErrorHandlerType } from 'amo/types/errorHandler';
+import type { I18nType } from 'amo/types/i18n';
+import type { DispatchFunc } from 'amo/types/redux';
+import type { ReactRouterMatchType } from 'amo/types/router';
+import { setViewContext } from 'amo/actions/viewContext';
+
+import './styles.scss';
+
+type Props = {||};
+
+type PropsFromState = {|
+  currentUser: UserType | null,
+  loading: boolean,
+  currentUser: UserType | null,
+  addonIdentifier: string,
```

Comment:
I don't think this is coming from the state. It's probably coming from React Router so more like: https://github.com/mozilla/addons-frontend/blob/fe7c0b08f2da598212854bd9eeae0e3b5b72de72/src/amo/pages/Block/index.js#L27-L30